src/client/headers/client/sqf/sqf.hpp

```diff
@@ -55,9 +55,9 @@ class __inline_sqf_helper_launcher {
             _sqf = std::regex_replace(_sqf, escape, "$1");
             const auto sqf_fnc = intercept::sqf::compile(_sqf);
             if (_capture_return) {
-                *_capture_return = intercept::sqf::call(sqf_fnc, _capture_args);
+                *_capture_return = intercept::sqf::call<game_value>(sqf_fnc, _capture_args);
```

Comment:
This shouldn't be needed anymore right?
```suggestion
                *_capture_return = intercept::sqf::call(sqf_fnc, _capture_args);
```